devlib/utils/android.py

```diff
@@ -231,12 +231,17 @@ def methods(self):
             parser = etree.XMLParser(encoding='utf-8', recover=True)
             xml_tree = etree.parse(StringIO(dump), parser)
 
-            package = next((i for i in xml_tree.iter('package')
-                           if i.attrib['name'] == self.package), None)
+            package = None
+            for i in xml_tree.iter('package'):
+                if i.attrib['name'] == self.package:
+                    package = next((i for methods in i.iter('method')
+                                   if methods.attrib['name'].startswith('test')), None)
```

Comment:
What is the reason or only picking up methods that begin with test? This method should be generic enough to extract all available methods. Is it possible to filter where this information is consumed?

Reply:
Hi Marc,
I was just getting the package which includes methods starts with the 'test' with other methods inside that package. Because we need 'test' methods to run the workload, but as you said to keep it generic, I will collect all the methods. WA will get the methods starts with the 'test' anyway.